code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -637,6 +638,10 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 				boutput(user, "<span class='notice'>You fill [src] with [amt] units of [target].</span>")
 				F.group.drain(F, amt / F.group.amt_per_tile, src) // drain uses weird units
 
+			if (is_sealed)
```

Comment:
This makes the following `else` trigger on a differen condition that before. That doesn't seem intended. Why not just put a single sealed check at the very beginning of the proc instead anyway?